tests/results_parser/results_parser_test.py

```diff
@@ -209,6 +211,7 @@ def test_parses_hosts_and_ports():
                     ],
                     "ports": [
                         {
+                            "port_str": "22",
```

Comment:
I don't get this, why do we need port_id and port_str?

Reply:
port_str forces it to be a string so we don't have commas in the results output for open ports.
port_id is left there for anyone that's already using it (e.g. secondary scanners), as well as if at some point someone wants to do a range search by port

Reply:
port_str forces it to be a string so we don't have commas in the results output for open ports.
port_id is left there for anyone that's already using it (e.g. secondary scanners), as well as if at some point someone wants to do a range search by port

Reply:
Where were the commas coming from? Can we have a conversation about this one?